service/src/main/java/bio/terra/workspace/service/workspace/CloudSyncRoleMapping.java

```diff
@@ -71,6 +71,7 @@ public class CloudSyncRoleMapping {
               "artifactregistry.tags.update",
               "cloudbuild.builds.create",
               "cloudbuild.builds.update",
+              "compute.instances.get",
```

Comment:
I think this just gives read access to metadata and not instance data which is fine, but we should test once this is merged